diff --git a/Makefile.common b/Makefile.common
index 57dabf5..8745901 100644
--- a/Makefile.common
+++ b/Makefile.common
@@ -349,0 +350,8 @@ OBJ += \
+ifeq ($(HAVE_AERA), 1)
+   DEFINES += -DHAVE_AERA
+   OBJ += aera/aera_platform.o \
+          gfx/drivers/aera_gfx.o \
+          input/drivers/aera_input.o \
+          audio/drivers/aera_audio.o
+endif
+
diff --git a/audio/audio_driver.c b/audio/audio_driver.c
index a4a28cf..2d442be 100644
--- a/audio/audio_driver.c
+++ b/audio/audio_driver.c
@@ -105,0 +106,3 @@ audio_driver_t *audio_drivers[] = {
+#ifdef HAVE_AERA
+   &audio_aera,
+#endif
diff --git a/audio/audio_driver.h b/audio/audio_driver.h
index cd0ca56..6bf5f93 100644
--- a/audio/audio_driver.h
+++ b/audio/audio_driver.h
@@ -429,0 +430 @@ extern audio_driver_t audio_audioworklet;
+extern audio_driver_t audio_aera;
diff --git a/gfx/video_driver.c b/gfx/video_driver.c
index 7917922..c9f6614 100644
--- a/gfx/video_driver.c
+++ b/gfx/video_driver.c
@@ -344,0 +345,3 @@ const video_driver_t *video_drivers[] = {
+#ifdef HAVE_AERA
+   &video_aera,
+#endif
diff --git a/gfx/video_driver.h b/gfx/video_driver.h
index ca6434c..da7fa66 100644
--- a/gfx/video_driver.h
+++ b/gfx/video_driver.h
@@ -1367,0 +1368 @@ extern video_driver_t video_network;
+extern video_driver_t video_aera;
diff --git a/input/input_driver.c b/input/input_driver.c
--- a/input/input_driver.c
+++ b/input/input_driver.c
@@ -297,0 +298,3 @@ input_device_driver_t *joypad_drivers[] = {
+#ifdef HAVE_AERA
+   &aera_joypad,
+#endif
@@ -302,0 +303,3 @@ input_driver_t *input_drivers[] = {
+#ifdef HAVE_AERA
+   &input_aera,
+#endif
diff --git a/input/input_driver.h b/input/input_driver.h
--- a/input/input_driver.h
+++ b/input/input_driver.h
@@ -1205,0 +1206,2 @@ extern input_driver_t input_test;
+extern input_driver_t input_aera;
+extern input_device_driver_t aera_joypad;
